modules/navier_stokes/tests/ins/open_bc/RZ_do_nothing_bcs_cone.i

```diff
@@ -0,0 +1,140 @@
+[GlobalParams]
+  rho = 1
+  mu = 1
+  integrate_p_by_parts = false
+  gravity = '0 0 0'
+  coord_type = RZ
+[]
+
+[Mesh]
+  file = '2d_cone.msh'
+[]
+
+[Problem]
+[]
+
+[Preconditioning]
+  [./SMP_PJFNK]
+    type = SMP
+    full = true
+    solve_type = Newton
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+  petsc_options_iname = '-pc_type -pc_factor_shift_type -pc_factor_shift_amount -ksp_type'
+  petsc_options_value = 'lu NONZERO 1.e-10 preonly'
```

Comment:
Good point. Unfortunately, most of my physics work so far has been with small problems so I've always used direct Newton. I don't have much experience with the best iterative solution options. I know that with `integrate_p_by_parts = true`, this new test does not pass using the options from velocity_inletBC.i